docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
+
+![Terminology process overview](https://www.lucidchart.com/invitations/accept/2eee45ed-5e62-4636-9a44-24b4414f273f)
```

Comment:
We may want to iterate on the chart to simplify it somewhat. It presently integrates the steps with the systems supporting those steps, but it may be simpler to simply tag each step with the system it interacts with (GitHub, spreadsheet, etc.) – possibly using an icon for the system.

Reply:
We can do this after the text is finalized though.

Reply:
I agree to finalize the chart later. It currently helps me to keep track of the details :smile: